_includes/newsletter.html

```diff
@@ -1,24 +1,15 @@
-<div class="guides-section-white">
-    <div class="row no-padding">
-        <div class="col-md-6 col-md-offset-3 newsletter text-center" >
-
+<div class="row">
+    <div class="guides-newsletter center">
+        <div class="col-md-6 modal-box text-center">
             <img src="/assets/img/guides/layout/img-newsletter.png" alt="newsletter_image" class="img-center"><br>
             <label>Newsletter</label>
             <p>Subscribe to the Gruntwork Newsletter to get notified when updates and new guides are released. No spam. Ever.</p>
-            <form id="newsletter-form" class="col-xs-10 col-xs-offset-1" method="post" action="//gruntwork.us13.list-manage.com/subscribe/post?u=8ccaad0747ac13944cf318e0f&amp;id=786bd365d1">
-                <div class="input-group">
-                    <input type="text" id="email" class="form-control guides-input-box" placeholder="Your Email Address">
-                    <div class="input-group-btn">
-                        <div class="btn-group" role="group"> 
-                            <button type="button" id="newsletter-subscribe" class="btn btn-primary" ga-event-category="newsletter" ga-event-action="nl-subscribe">
-                                Subscribe
-                            </button>
-                        </div>
-                    </div>
+            <form  id="guides-newsletter-form"class="col-xs-10 col-xs-offset-1" method="post" action="//gruntwork.us13.list-manage.com/subscribe/post?u=8ccaad0747ac13944cf318e0f&amp;id=c19e11ead2" target="_blank">
```

Comment:
I noticed the `id=...` param changed. Is that intentional?

Reply:
Yes, it was. I wanted to specify that this is coming from the guides page. In case we want to make reference to it anytime.